plugins/copr.py

```diff
@@ -223,6 +223,12 @@ def run(self):
                   'copr command are required'))
         try:
             chroot = self.opts.arg[1]
+            if len(self.opts.arg) > 2:
+                raise dnf.exceptions.Error(_('Too many arguments.'))
+            chroot_parts = chroot.split("-")
+            if len(chroot_parts) < 3:
+                raise dnf.exceptions.Error(_('Bad format of optional chroot. Format is '
+                                             'distribution-version-architecture.'))
```

Comment:
Please add a definite particle for the format in the message:
> Bad format of optional chroot. **The** format is ...

Also, why not parse and store the `short_chroot` and `arch` in `self` here, so that you don't need to split the `chroot` for the second time later on? It's a bit cleaner to do all the validation and processing of input in one place and then just use those values.